modules/packages/LinearAlgebra.chpl

```diff
@@ -1757,6 +1757,261 @@ proc svd(A: [?Adom] ?t) throws
   return (u, s, vt);
 }
 
+pragma "no doc" 
+proc svd(A : [?Adom] ?eltType, iterations : int = 30, eps : real = 1e-8)
```

Comment:
These news args will need to be documented. Maybe we want 2 separate documented functions for svd since their interface doesn't 100% match.

Reply:
I would be in favor of having separate documentations until both interfaces can be unified